src/test/java/nextstep/ladder/domain/CrossingPointTest.java

```diff
@@ -0,0 +1,83 @@
+package nextstep.ladder.domain;
+
+import nextstep.ladder.strategy.DirectionStrategy;
+import nextstep.ladder.strategy.RandomDirectionStrategy;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.BDDAssertions.then;
+
+public class CrossingPointTest {
+    private CrossingPoint connectableCrossingPoint;
+    private CrossingPoint unConnectableCrossingPoint;
+
+    @BeforeEach
+    void setUp() {
+        connectableCrossingPoint = CrossingPoint.first(connectableTrueStrategy);
+        unConnectableCrossingPoint = CrossingPoint.first(connectableFalseStrategy);
+    }
+
+    private DirectionStrategy connectableTrueStrategy = new RandomDirectionStrategy() {
+        @Override
+        public boolean isConnectable() {
+            return true;
+        }
+    };
+
+    private DirectionStrategy connectableFalseStrategy = new RandomDirectionStrategy() {
+        @Override
+        public boolean isConnectable() {
+            return false;
+        }
+    };
```

Comment:
```suggestion
    private DirectionStrategy connectableTrueStrategy = () -> true;
    private DirectionStrategy connectableFalseStrategy = () -> false;
```
이렇게 간단하게 표현할 수 있습니다.

Reply:
@JunilHwang
해당 부분 `DirectionStrategy`가 이전에는 `isConnectable()` 하나여서 말씀하신 방식대로 구현 했었는데
생성로직까지 모두 추가하게 된 후, 메소드가 3개가 되어 `() -> true`를 사용하지 못해서 위에 방식처럼 표현하게 되었습니다!